packages/nuxt/src/core/templates.ts

```diff
@@ -366,6 +366,7 @@ export const nitroSchemaTemplate: NuxtTemplate = {
     return /* typescript */`
 ${lines.join('\n')}
 /// <reference path="./runtime-config.d.ts" />
+/// <reference path="./shared.d.ts" />
```

Comment:
```suggestion
/// <reference path="./types/shared.d.ts" />
```

i would also recommend renaming to shared-imports.d.ts for clarity